Shoko.Server/API/v3/Controllers/DashboardController.cs

```diff
@@ -105,16 +105,17 @@ private static bool MissingBothTvDBAndMovieDBLink(SVR_AnimeSeries ser)
         [HttpGet("TopTags")]
         public List<Tag> GetTopTags()
         {
-            return GetTopTags(10);
+            return GetTopTags();
```

Comment:
This will cause infinite recursion. The 10 is passing a default value to the other endpoint. I didn't know how to do optional routes, so this is how I did it at the time.